var/da/da_chem_sfc/da_transform_xtoy_chem_sfc_adj.inc

```diff
@@ -23,25 +23,26 @@ subroutine da_transform_xtoy_chem_sfc_adj(grid, iv, jo_grad_y, jo_grad_x)
 
    if (sfc_assi_options == sfc_assi_options_1) then
 
```

Comment:
Same comment as for tangent linear code. Can remove this line.

Reply:
Agree